src/ucp/proto/proto_multi.inl

```diff
@@ -268,7 +267,8 @@ ucp_proto_eager_bcopy_multi_common_send_func(
         pack_cb  = pack_cb_middle;
         hdr_size = hdr_size_middle;
     }
-    pack_ctx.max_payload = ucp_proto_multi_max_payload(req, lpriv, hdr_size);
+    pack_ctx.max_payload = ucp_proto_multi_max_payload(req, lpriv, hdr_size,
```

Comment:
maybe we can just add hdr_size to req->send.state.dt_iter.length ?

Reply:
From existing implementation perspective it is more consistent to keep hrd_size out of req->send.state.dt_iter.length. E.g. user header is sent within first send of multi fragment send.